concepticondata/concepticon.tsv

```diff
@@ -2247,7 +2247,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2246	BELL	Basic actions and technology	A simple sound-making device.	Person/Thing	
 2247	CATERPILLAR	Animals	The larvae of members of the order Lepidoptera (the insect order comprising butterflies and moths).	Person/Thing	
 2248	KNOW (SOMEBODY)	Cognition	To have met and recognize somebody.	Action/Process	
-2249	THIN (OF SHAPE OF OBJECT)	Spatial relations	Being narrow in size.	Property	
+2249	THIN (OF SHAPE OF OBJECT)	Spatial relations	Being very narrow (of flat or long objects).	Property	
```

Comment:
Why 'very' narrow? This is usually elicited in plain oposition with 'thick', right? It is not necessarily 'very' narrow

Reply:
All the other "thin" concepts already had the "very" in their definitions. But I guess it's best to remove it from all of them.

Reply:
On second thought, I will remove the "very" only from THIN and from THIN (OF SHAPE OF OBJECT).